Apps/W1/HybridGP/app/src/Migration/Support/WizardIntegration.codeunit.al

```diff
@@ -74,6 +76,9 @@ codeunit 4035 "Wizard Integration"
         DataSyncStatus.ParsePosting();
         HelperFunctions.PostGLTransactions();
         HelperFunctions.SetProcessesRunning(false);
+
+        if GPCompanyAdditionalSettings.GetMigrateHistory() then
+            HybridCloudManagement.CreateAndScheduleBackgroundJob(Codeunit::"GP Populate Hist. Tables", 'Migrate GP Historical Snapshot');
```

Comment:
This method is nice, however you can create your own. We could also change the CreateAndScheduleBackgroundJob to allow more parameters.
We should be able to specify the different "Maximum No. of Attempts to Run"  and also the timeout  to allow this one to run for more than 12h.